contents/blog/data-management-feature.md

```diff
@@ -0,0 +1,95 @@
+---
+date: 2022-03-23
+title: Introducing Data Management in PostHog
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+author: ["alex-kim"]
+categories: ["Product updates", "Launch week"]
+---
+
+As PostHog grows, the data that users are sending to PostHog continues to grow into the double-digit billions. In just the last year (since we started measuring total events), we've gone from ~8B to ~36.5B total events ingested in PostHog Cloud!
+
+![PostHog - total events](../images/blog/data-management-feature/total_events.png)
+
+Data explosion is real and can be a nightmare to scale in the context of a product analytics tool. Furthermore, poor data management can seriously harm your business as it can lead to inaccurate analyses and poor business decisions.
```

Comment:
```suggestion
This is great for us as a business, but data explosion is real and can be a nightmare to scale in the context of a product analytics tool. Furthermore, poor data management can seriously harm your business as it can lead to inaccurate analyses and poor business decisions.
```